invalid memory address or nil pointer dereference when a secretfile is empty

The current code doesn't check if a secret file exists and isn't empty.

```
> talhelper genconfig
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x0 pc=0x10383033c]

goroutine 1 [running]:
github.com/budimanjojo/talhelper/pkg/talos.NewClusterInput(0x14000456408, {0x10386b0cc, 0x13}, {0x103859c80, 0x5})
    github.com/budimanjojo/talhelper/pkg/talos/input.go:44 +0x1cc
github.com/budimanjojo/talhelper/pkg/generate.GenerateConfig(0x14000456408, 0x0, {0x1038661fa, 0xf}, {0x10386b0cc?, 0x1c?}, {0x103859c80, 0x5}, 0x0)
    github.com/budimanjojo/talhelper/pkg/generate/config.go:23 +0x54
github.com/budimanjojo/talhelper/cmd.init.func7(0x140002b6c00?, {0x104632440?, 0x4?, 0x10385930d?})
    github.com/budimanjojo/talhelper/cmd/genconfig.go:50 +0x2a4
github.com/spf13/cobra.(*Command).execute(0x1045b7ec0, {0x104632440, 0x0, 0x0})
    github.com/spf13/cobra@v1.8.0/command.go:987 +0x828
github.com/spf13/cobra.(*Command).ExecuteC(0x1045b9000)
    github.com/spf13/cobra@v1.8.0/command.go:1115 +0x344
github.com/spf13/cobra.(*Command).Execute(...)
    github.com/spf13/cobra@v1.8.0/command.go:1039
github.com/budimanjojo/talhelper/cmd.Execute(...)
    github.com/budimanjojo/talhelper/cmd/root.go:61
main.main()
    github.com/budimanjojo/talhelper/main.go:10 +0x28
```

--- a/pkg/talos/input.go
+++ b/pkg/talos/input.go
@@ -31,6 +31,9 @@ func NewClusterInput(c *config.TalhelperConfig, secretFile string, mode string) 
 		if err != nil {
 			return nil, err
 		}
+		if len(decrypted) == 0 {
+			return nil, fmt.Errorf("secret file %s is empty", secretFile)
+		}
 
 		decrypted, err = substitute.SubstituteEnvFromByte(decrypted)
 		if err != nil {

